apps/PCLWrapper/pcl/registration/impl/super4pcs.hpp

```diff
@@ -77,7 +77,7 @@ pcl::Super4PCS<PointSource, PointTarget>::computeTransformation (PointCloudSourc
   fillPointSet(*target_, set1);
   fillPointSet(*input_, set2);;
 
-  float score = matcher.ComputeTransformation(set1, set2, final_transformation_, sampler, visitor);
+  fitness_score_ = matcher.ComputeTransformation(set1, set2, final_transformation_, sampler, visitor);
```

Comment:
at least this could be merged as a bug fix independently right?

Reply:
That's correct, I created a separate PR #65.